autogen/main/cluster.tf.tmpl

```diff
@@ -388,6 +388,14 @@ resource "google_container_cluster" "primary" {
       }
     }
 
+    dynamic "parallelstore_csi_driver_config" {
```

Comment:
I don't think we need to add for Autopilot clusters as for them it's enabled by default?

Reply:
will check

Reply:
<img width="965" alt="Screenshot 2024-12-16 at 18 18 28" src="https://github.com/user-attachments/assets/a57aae54-9648-4204-ad2a-3f8c28ea8eac" />

Only found this as ref, will go with not setting this on autopilot for now